frontend/src/views/Game.tsx

```diff
@@ -147,8 +148,28 @@ function GamePage() {
     setTimeUp(newGame.gameTimer.timeUp);
   };
 
+  const createCodeLanguageArray = () => {
+    while (currentLanguage.length < problems.length) {
+      currentLanguage.push(Language.Python);
+    }
+
+    while (currentCode.length < problems.length) {
+      currentCode.push('');
+    }
+  };
+
+  createCodeLanguageArray();
+
+  const setOneCurrentLanguage = (newLanguage: Language) => {
+    currentLanguage[currentProblem.valueOf()] = newLanguage;
```

Comment:
is there a need to use valueOf here and below? i would have thought it would be automatic (maybe lowercase `number` instead of `Number` could help? idk)

Reply:
Yeah ok, I'll change that in my next PR.